app/assets/javascripts/short-form/components/preference-component.html.slim

```diff
@@ -1,14 +1,73 @@
 .checkbox-block ng-class="{'margin-bottom': $ctrl.marginBottom}"
-  preference-checkbox(application="$ctrl.application" preference="{{$ctrl.preference}}" invalid="$ctrl.inputInvalid($ctrl.preference)" title="{{$ctrl.title}}" translated-description="{{$ctrl.translatedDescription}}" custom-description="{{$ctrl.customDescription}}" short-description="{{$ctrl.translatedShortDescription}}" on-change="$ctrl.onChange()" required="$ctrl.required()")
-
-  .form-group.input-inset ng-if="$ctrl.application.preferences[$ctrl.preference]"
+  preference-checkbox(
+    application="$ctrl.application"
+    preference="{{$ctrl.preference}}"
+    invalid="$ctrl.inputInvalid($ctrl.preference)"
+    title="{{$ctrl.title}}"
+    translated-description="{{$ctrl.descriptionToTranslate()}}"
+    custom-description="{{$ctrl.customDescription}}"
+    short-description="{{$ctrl.translatedShortDescription}}"
+    read-more-url="{{::$ctrl.readMoreUrl}}"
+    on-change="$ctrl.onChange()"
+    required="$ctrl.required()"
+  )
+  .form-group.input-inset.margin-bottom-none.padding-bottom-none ng-if="$ctrl.application.preferences[$ctrl.preference]"
     / member selector
-    preference-member-selector(application="$ctrl.application" preference="{{$ctrl.preference}}" eligible-members="$ctrl.eligibleMembers" label="{{$ctrl.memberSelectorLabel}}")
+    preference-member-selector(
+      application="$ctrl.application"
+      preference="{{::$ctrl.preference}}"
```

Comment:
looks like I also added :: to this line. I'm going to remove it because there's a possibility that it could cause issues